backend/tests/verify-bootstrap-import.js

```
@@ -0,0 +1,20 @@
+  '../src/infra/database/migrations/bootstrap/bootstrap-migrations.js'
+);
+
+if (!fs.existsSync(bootstrapPath)) {
+  throw new Error(`Bootstrap migration script not found at: ${bootstrapPath}`);
+}
+
+if (typeof logger.info !== 'function') {
+  throw new Error('Failed to resolve bootstrap logger dependency via tsx path aliases');
+}
+
+console.log('Bootstrap import path OK');
```

ESLint

[error] 'console' is not defined.

(no-undef)